scanner/src/main/resources/META-INF/services/org.ossreviewtoolkit.scanner.ScannerFactory

```diff
@@ -3,3 +3,5 @@ org.ossreviewtoolkit.scanner.scanners.BoyterLc$Factory
 org.ossreviewtoolkit.scanner.scanners.FileCounter$Factory
 org.ossreviewtoolkit.scanner.scanners.Licensee$Factory
 org.ossreviewtoolkit.scanner.scanners.ScanCode$Factory
+org.ossreviewtoolkit.scanner.scanners.ScanOSS$Factory
```

Comment:
Also here, the class should be named `ScanOss` to follow Java naming conventions.